geode-core/src/distributedTest/java/org/apache/geode/internal/cache/DeltaForceSizingFlagDUnitTest.java

```diff
@@ -0,0 +1,995 @@
+    });
+  }
+
+  private boolean prHostsBucketForKey(MemberVM memberVM, final Object key) {
+    Boolean result = (Boolean) memberVM.invoke("prHostsBucketForKey", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      DistributedMember myId = cache.getDistributedSystem().getDistributedMember();
+      Region region = cache.getRegion("region");
+      DistributedMember hostMember = PartitionRegionHelper.getPrimaryMemberForKey(region, key);
+      if (hostMember == null) {
+        throw new IllegalStateException("bucket for key " + key + " is not hosted!");
+      }
+      boolean res = Boolean.valueOf(myId.equals(hostMember));
+      return res;
+    });
+    return result.booleanValue();
+  }
+
+  private void put(VM vm0, final Object key, final Object value) {
+    vm0.invoke("Put data", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      LocalRegion region = (LocalRegion) cache.getRegion("region");
+      region.put(key, value);
+    });
+  }
+
+  private void get(VM vm0, final Object key, final Object value) {
+    vm0.invoke("Get data", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      LocalRegion region = (LocalRegion) cache.getRegion("region");
+      assertEquals(value, region.get(key));
+    });
+  }
+
+  protected static int getObjectSizerInvocations(LocalRegion region) {
+    TestObjectSizer sizer = (TestObjectSizer) region.getEvictionAttributes().getObjectSizer();
+    int result = sizer.invocations.get();
+    region.getCache().getLogger().info("objectSizerInvocations=" + result);
+    return result;
+  }
+
+  private static long getSizeFromEvictionStats(LocalRegion region) {
+    long result = region.getEvictionCounter();
+    return result;
+  }
+
+  private static long getEvictions(LocalRegion region) {
+    return region.getTotalEvictions();
+  }
+
+  private void createRR(MemberVM memberVM) {
+    memberVM.invoke("Create replicateRegion", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      AttributesFactory<Integer, TestDelta> attr = new AttributesFactory<Integer, TestDelta>();
+      attr.setDiskSynchronous(true);
+      attr.setDataPolicy(DataPolicy.REPLICATE);
+      attr.setScope(Scope.DISTRIBUTED_ACK);
+      attr.setEvictionAttributes(EvictionAttributes.createLRUMemoryAttributes(1,
+          new TestObjectSizer(), EvictionAction.OVERFLOW_TO_DISK));
+      attr.setDiskDirs(getMyDiskDirs());
+      Region region = cache.createRegion("region", attr.create());
+    });
+  }
+
+  private void assertValueType(VM vm, final Object key, final ValueType expectedType) {
+    vm.invoke("assertValueType", () -> {
+      Cache cache = ClusterStartupRule.getCache();
+      LocalRegion region = (LocalRegion) cache.getRegion("region");
+      Object value = region.getValueInVM(key);
+      switch (expectedType) {
+        case RAW_VALUE:
+          assertTrue("Value was " + value + " type " + value.getClass(),
+              !(value instanceof CachedDeserializable));
```

Comment:
This assert can be simplified to `assertFalse()` and removing the negation from the condition being asserted on.

Reply:
Done.